### ISSUE TITLE
Alacritty rule for floating makes window appear always on Monitor 1

### ISSUE THREAD

**Issue by blumberg** [baskerville/bspwm#1396]
When setting a rule for **alacritty** to open as floating, it always opens on Monitor 1 (doesn't accept `center` neither the position section of `rectangle`).
If I have monitor 2 selected the window doesn't get focus until I click on it.

This is my rule:
```shell
bspc rule -a Alacritty follow=on state=floating center=true
```

I've tried with and without `follow` argument and with and without `center` argument and the results is always the same.

### System

**Distro:** Arch
**bspwm:** 0.9.10

**Comment by blumberg** [comment]
Yes, all my other rules applies normally. I've tried `Yubico Authenticator`, `Imager`, `XCalc` and `Mousepad` and they all worked fine.

**Comment by blumberg** [comment]
I saw that thread but as for me the rule was applying (alacritty was opening as floating) i haven't follow the script.

Now I gave it a shot and something even weird happened. When I set up `external_rules_command` (even if the script is empty` alacrity works as intended and opens centred. If I return `external_rules_command` to an empty string, the old (wrong) behaviour returns.

Having external rules setup to anything **did solve my problem**, but looks more like a workaround then a proper solving, as both actions shouldn't happen. Alacritty should follow the center rule even though there is no external rules, and adding external rules shouldn't solve my issue.

If you want me to close this thread I can do. If you want me to help debugging and find the root cause I'm open to it as well

**Comment by zjeffer** [comment]
This is a very strange issue to me, I don't know enough about the inner workings of bspwm, so I recommend you keep this issue open and wait for someone smarter than me to help you out ;)
